velox/vector/BaseVector.cpp

```diff
@@ -544,6 +544,12 @@ std::string BaseVector::toString(vector_size_t index) const {
   return out.str();
 }
 
+std::string BaseVector::toString(
+    vector_size_t index,
+    const VectorCastOptions& options) const {
```

Comment:
Is it better to use a function to process all possible vectors, to avoid introducing extra API to each implementation?